openapi/src/test/groovy/io/micronaut/openapi/proto/MyEnum.java

```diff
@@ -105,7 +105,7 @@ public static MyEnum valueOf(
 
   private final int value;
 
-  private MyEnum(int value) {
```

Comment:
why is this change necessary in the PR to update versions?

Reply:
not necessary

Reply:
reverted